src/__tests__/articles/Article.test.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
```

Comment:
@djallas, would you put tests of components related to articles in the folder of components

Reply:
Sure